packages/terra-date-picker/src/react-datepicker/stylesheets/react_datepicker.module.scss

```diff
@@ -8,6 +8,7 @@
 
 :local {
   .react-datepicker {
+    align-self: flex-start;
```

Comment:
why do we need align property change here..??

Reply:
So, with only scroll property screen was blocking access to the top elements i.e months , year (due  property of parent element). Like this:- 

https://github.com/cerner/terra-framework/assets/126875089/14d0241e-f765-408a-a4ac-57640d2423dd

To handle this scenario require to add align property.